packages/turf-standard-deviational-ellipse/index.js

```diff
@@ -0,0 +1,133 @@
+ * var points = turf.randomPoint(400, {bbox: bbox});
+ * var sdEllipse = turf.standardDeviationalEllipse(points, options);
+ *
+ * //addToMap
+ * var addToMap = [points, sdEllipse];
+ *
+ */
+function standardDeviationalEllipse(points, options) {
+    // Optional params
+    options = options || {};
+    if (!isObject(options)) throw new Error('options is invalid');
+    var steps = options.steps || 64;
+    var weightTerm = options.weight;
+    var properties = options.properties || {};
+
+    // Validation:
+    if (!isNumber(steps)) throw new Error('steps must be a number');
+    if (!isObject(properties)) throw new Error('properties must be a number');
+
+    // Calculate mean center & number of features:
+    var numberOfFeatures = coordAll(points).length;
+    var meanCenter = centerMean(points, {weight: weightTerm});
+
+    // Calculate angle of rotation:
+    // [X, Y] = mean center of all [x, y].
+    // theta = arctan( (A + B) / C )
+    // A = sum((x - X)^2) - sum((y - Y)^2)
+    // B = sqrt(A^2 + 4(sum((x - X)(y - Y))^2))
+    // C = 2(sum((x - X)(y - Y)))
+
+    var xDeviationSquaredSum = 0;
+    var yDeviationSquaredSum = 0;
+    var xyDeviationSum = 0;
+
+    featureEach(points, function (point) {
+        var weight = point.properties[weightTerm] || 1;
+        var deviation = getDeviations(getCoords(point), getCoords(meanCenter));
+        xDeviationSquaredSum += Math.pow(deviation.x, 2) * weight;
+        yDeviationSquaredSum += Math.pow(deviation.y, 2) * weight;
+        xyDeviationSum += deviation.x * deviation.y * weight;
+    });
+
+    var bigA = xDeviationSquaredSum - yDeviationSquaredSum;
+    var bigB = Math.sqrt(Math.pow(bigA, 2) + 4 * Math.pow(xyDeviationSum, 2));
+    var bigC = 2 * xyDeviationSum;
+    var theta = Math.atan((bigA + bigB) / bigC);
+    var thetaDeg = theta * 180 / Math.PI;
+
+    // Calculate axes:
+    // sigmaX = sqrt((1 / n - 2) * sum((((x - X) * cos(theta)) - ((y - Y) * sin(theta)))^2))
+    // sigmaY = sqrt((1 / n - 2) * sum((((x - X) * sin(theta)) - ((y - Y) * cos(theta)))^2))
+    var sigmaXsum = 0;
+    var sigmaYsum = 0;
+    var weightsum = 0;
+    featureEach(points, function (point) {
+        var weight = point.properties[weightTerm] || 1;
+        var deviation = getDeviations(getCoords(point), getCoords(meanCenter));
+        sigmaXsum += Math.pow((deviation.x * Math.cos(theta)) - (deviation.y * Math.sin(theta)), 2) * weight;
+        sigmaYsum += Math.pow((deviation.x * Math.sin(theta)) + (deviation.y * Math.cos(theta)), 2) * weight;
+        weightsum += weight;
+    });
+
+    var sigmaX = Math.sqrt(2 * sigmaXsum / weightsum);
+    var sigmaY = Math.sqrt(2 * sigmaYsum / weightsum);
+
+    var theEllipse = ellipse(meanCenter, sigmaX, sigmaY, {units: 'degrees', angle: thetaDeg, steps: steps, properties: properties});
+    var pointsWithinEllipse = pointsWithinPolygon(points, featureCollection([theEllipse]));
+    var standardDeviationalEllipseProperties = {
```

Comment:
We might want to define these properties in the Typescript definition, that way it's the GeoJSON properties are documented, similar to [`@turf/clusters-dbscan`](https://github.com/Turfjs/turf/blob/master/packages/turf-clusters-dbscan/index.d.ts#L4-L8).